WcaOnRails/spec/rails_helper.rb

```diff
@@ -34,7 +34,12 @@
 # To debug feature specs using phantomjs, set `Capybara.javascript_driver = :poltergeist_debug`
 # and then call `page.driver.debug` in your feature spec.
 Capybara.register_driver :poltergeist_debug do |app|
-  Capybara::Poltergeist::Driver.new(app, inspector: true, phantomjs: Phantomjs.path, debug: true)
+  Capybara::Poltergeist::Driver.new(app, inspector: true, phantomjs: Phantomjs.path, debug: true, phantomjs_options: ['--ignore-ssl-errors=yes', '--ssl-protocol=any'])
```

Comment:
Would be great to have a comment for why we need each of these options =)

Reply:
Ah it's just below on the non-debug poltergeist driver!
But I can repeat it :)

Reply:
Yeah, I was thinking of a more precise note how each of the phantomjs options helps us ^^

Reply:
I've updated the comment!